### PR TITLE
Patch 2.21.1

### PR DESCRIPTION
**Pull request by mddilley**
## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/19538

This PR updates the Lexical `OnEditPlugin` to handle status updates created through the input in the `ProjectPhaseForm` which is not saved as rich text. **This patch will save non-rich text status updates as rich text even though they were not originally saved that way** which I think is fine since we are already primarily creating notes this way. I'll create an issue to see what adding the Lexical editor to this form would look like though wrapping it in Reach Hook Form could get bumpy.

## Testing
**URL to test:** 

https://deploy-preview-1459--atd-moped-main.netlify.app/moped/projects/

**Steps to test:**
1. Go to a project timeline in staging ([like this one](https://moped.austinmobility.io/moped/projects/2045?tab=timeline)) and create or edit a phase and add a status update before saving. This will create a new status update that you can see in the **Notes** tab.
Screenshot 2024-10-21 at 12 31 34 PM

2. Go to the **Notes** tab and edit the status update that you just created. Notice that the editor opens up but does not show text.
3. Now, do the same in this deploy preview. Notice that the content of the status update does display as expected when opening in the **Notes** tab.

---
#### Ship list
- [x] Code reviewed 
- [x] Product manager approved
- [x] Product manager checked [DB view dependencies](https://atd-dts.gitbook.io/moped-documentation/product-management/updating-the-arcgis-online-components-database-view)
- [x] Product manager added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable
   - Updated existing test to check if Phase status updates can be edited in the **Notes** tab

### CODE REVIEW

File: moped-editor/src/views/projects/projectView/ProjectNotes.js
Comment (mddilley) [inline moped-editor/src/views/projects/projectView/ProjectNotes.js:1]: these changes came from Prettier

**Review by johnclary — APPROVED** [review]
👍 i tested this through the deploy preview and the code change makes sense to me. this looks good to go 🚢

**Review by chiaberry — APPROVED** [review]
im glad you were able to find a fix so quickly 

🚤

**Comment by mddilley** [comment]
Thank y'all!!! I'm going to go ahead and merge this and then move these changes into `main` too.

**Pull request merged**